pkg/cloudprovider/providers/vsphere/vclib/datacenter.go

```diff
@@ -142,6 +143,23 @@ func (dc *Datacenter) GetVMMoList(ctx context.Context, vmObjList []*VirtualMachi
 	return vmMoList, nil
 }
 
+// GetVirtualDiskPage83Data gets the virtual disk UUID by diskPath
+func (dc *Datacenter) GetVirtualDiskPage83Data(ctx context.Context, diskPath string) (string, error) {
```

Comment:
Can we move this function to appropriate disk related file? Datacenter.GetVirtualDiskPage83Data doesn't look appropriate. Same for other new functions in this file?

Reply:
GetVirtualDiskPage83Data depends on vdm.QueryVirtualDiskUuid(ctx, diskPath, dc.Datacenter) which queries virtual disk UUID in a datacenter. Based on the discussion offline with @divyenpatel, we have decided to keep this function in this file only.

Reply:
it is kind of strange name?

Reply:
Nope, vsphere developers will be aware of this function name. All the vsphere docs use this name to get virtual disk UUID.